modules/ROOT/partials/configuration/mergetags_prefix.adoc

```diff
@@ -20,3 +20,5 @@ tinymce.init({
   mergetags_prefix: '{{'
 });
 ----
+
+IMPORTANT: Whatever character or characters are set as the Merge Tags prefix, it or they must be different to the characters set as the Merge Tags suffix. For example, if the Merge Tags prefix is set to `%`, the Merge Tags suffix cannot also be `%`.
```

Comment:
> Would this be better as its own partial? Simply one saying that the
> prefix and suffix should not be the same character/s,

I spent a chunk of time using this approach but abandoned it because I couldn’t come up with an elegant way of including a concrete example that didn’t give the impression that the implementation is about one thing (eg the suffix) not being the same as the other (eg the prefix).

Since the actual implementation is about the two strings being different, I don’t want to offer even a hint that it is anything else. (And, yes, the two notions are different. One being different to the other implies a causal chain that can and will lead readers to assume further things that are also untrue.)

There are inelegant ways aplenty to do this in one paragraph, BTW. None of them are easy to read.

> then included once on the Merge Tags page.

I spent a chunk of time on this as well. The clear spots for a single admonition are

* as an admonition immediately following the *Basic setup* section; or

* as a new item in the *Using Merge Tags* section.

* as an admonition immediately following the *Using Merge Tags* section.

Putting the single admonition after the *Basic setup* section means also re-writing some of that section so the idea of prefixes and suffixes is already introduced. And that reads like explaining the *Options* outside the *Options* section. Not really *Basic setup* material.

And, as a new item in the *Using Merge Tags* section, it doesn’t stand out enough. And it’s a note less about how to use the Tags as how to not use them.

And, as a single admonition after the *Using Merge Tags* section felt like a warning out of nowhere. It’s good to know this, but what does this warning have to do with formatting and nesting Merge Tags?

FWIW, I also wrote a single admonition that could be an `include` for both the `mergetags_prefix` and `mergetags_suffix` sections.

It had the same problems as my first approach noted above: it got incredibly wordy and inelegant and made providing a concrete example really messy.

FWIW, even after all this, I mostly agree that a single included partial would be better. And a full re-write and re-structure of this section would make such an approach absolutely the way to go.

I don’t, unfortunately, have the time or luxury for such an approach.